OracleWebCenterSites/README.md

```diff
@@ -0,0 +1,422 @@
+ 
+To download images from Oracle Container Registry (OCR) and Oracle Technology Network (OTN):
+
+Sign in to [OCR](https://container-registry.oracle.com). Click the **Sign in** link that's on the top-right of the Web page.
+
+### A. To download and set up Oracle Fusion Middleware Infrastructure image.
+1. Click **Home > Middleware** and then click **Continue** for the <i>fmw-infrastructure</i> repository.
+2. Click **Accept** to accept the license agreement.
+3. To download Oracle Fusion Middleware infrastructure base image:    
+    ```
+        $ docker pull container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3
+    ```
+	
+    Retag the Oracle Fusion Middleware Infrastructure Docker image by running the following command:
+    ```
+        $ docker tag container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3 oracle/fmw-infrastructure:12.2.1.3
+    ```
+    **Note**: 
+    - If you want to download image from the Docker store, see [FAQ](#7-alternate-download-location-for-oracle-fusion-middleware-infrastructure-and-oracle-database-images) section.
+    - If you want to build image from the GitHub, see [FAQ](#9-how-do-i-build-an-oracle-fusion-middleware-infrastructure-1221x-base-image) section.
+
+### B. To download and set up Oracle Database Enterprise Edition image.
+1. Click **Home > Database** and then click **Continue** for <i>enterprise</i> repository.
+2. Click **Accept** to accept the license agreement.
+3. From the terminal, execute the following commands:
+       
+    Download Oracle Database 12.2.0.1 base image:    
+    ``` 
+        $ docker pull container-registry.oracle.com/database/enterprise:12.2.0.1    
+    ```
+    Retag the Oracle Database Docker image by running the following command:
+    ```
+       $ docker tag container-registry.oracle.com/database/enterprise:12.2.0.1 database/enterprise:12.2.0.1
+    ```
+    **Note**: 
+    - If you want to download image from the Docker store, see [FAQ](#7-alternate-download-location-for-oracle-fusion-middleware-infrastructure-and-oracle-database-images) section.
+    - If you want to build image from the GitHub, see [FAQ](#8-how-do-i-build-an-oracle-database-1221x-base-image) section.
+	
+	
+### C. To download Oracle WebCenter Sites Docker files and binary file.
+
+1. Download the [docker-images-master](https://github.com/oracle/docker-images/archive/master.zip) zip file. The zip file contains Docker files and samples to build Docker images for Oracle products and open source projects.
+2. Download Oracle WebCenter Sites 12c R2 12.2.1.3 binary from [OTN](http://www.oracle.com/technetwork/middleware/webcenter/sites/downloads/index.html).
+3. Save the Oracle WebCenter Sites 12.2.1.3 binary at this location: `../docker-images-master/OracleWebCenterSites/dockerfiles/12.2.1.3/`.
+
+## 5. Building Oracle WebCenter Sites Docker Images
+1. Go to the folder where Docker scripts are located:
+```
+    $ cd ../docker-images-master/OracleWebCenterSites/dockerfiles
```

Comment:
I think you should use an relative path based off where the root of the clone of the repo is, because the end-user can clone anywhere on their filesystem.

Reply:
Changes incorporated into latest file.